10월 18, 2014 7:27:04 오후 com.microsoft.reef.runtime.common.evaluator.context.defaults.DefaultContextStartHandler onNext
정보: DefaultContextStartHandler received: com.microsoft.reef.runtime.common.evaluator.context.ContextStartImpl@398e4cb
10월 18, 2014 7:27:06 오후 com.microsoft.reef.io.network.naming.NameLookupClient remoteLookup
정보: Looking up ControllerTask on NameServer /169.254.245.8:5678
10월 18, 2014 7:27:06 오후 com.microsoft.reef.runtime.common.evaluator.context.ContextRuntime close
경고: Shutting down a task because the underlying context is being closed.
10월 18, 2014 7:27:06 오후 com.microsoft.reef.runtime.common.evaluator.task.TaskRuntime close
경고: Trying to close a task that is in state: DONE. Ignoring.
10월 18, 2014 7:27:06 오후 com.microsoft.reef.runtime.common.evaluator.context.defaults.DefaultContextStopHandler onNext
정보: DefaultContextStopHandler received: com.microsoft.reef.runtime.common.evaluator.context.ContextStopImpl@12a9a994
10월 18, 2014 7:27:06 오후 com.microsoft.wake.remote.transport.netty.NettyChannelHandler exceptionCaught
정보: Unexpected exception from downstream. channel: [id: 0xea04225c, /169.254.245.8:3779 => /169.254.245.8:7003] local: /169.254.245.8:7003 remote: /169.254.245.8:3779
10월 18, 2014 7:27:06 오후 com.microsoft.wake.remote.transport.netty.NettyChannelHandler exceptionCaught
경고: Unexpected exception from downstream.
java.io.IOException: 현재 연결은 원격 호스트에 의해 강제로 끊겼습니다
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379)
	at io.netty.buffer.UnpooledUnsafeDirectByteBuf.setBytes(UnpooledUnsafeDirectByteBuf.java:446)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:879)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:225)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:114)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
	at java.lang.Thread.run(Thread.java:745)

10월 18, 2014 7:27:06 오후 com.microsoft.wake.remote.transport.netty.NettyChannelHandler exceptionCaught
정보: Unexpected exception from downstream. channel: [id: 0x30b0ea90, /169.254.245.8:3782 => jw-PC/169.254.245.8:7000] local: /169.254.245.8:3782 remote: jw-PC/169.254.245.8:7000
10월 18, 2014 7:27:06 오후 com.microsoft.wake.remote.transport.netty.NettyChannelHandler exceptionCaught
경고: Unexpected exception from downstream.
java.io.IOException: 현재 연결은 원격 호스트에 의해 강제로 끊겼습니다
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379)
	at io.netty.buffer.UnpooledUnsafeDirectByteBuf.setBytes(UnpooledUnsafeDirectByteBuf.java:446)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:879)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:225)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:114)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
	at java.lang.Thread.run(Thread.java:745)

10월 18, 2014 7:27:06 오후 com.microsoft.wake.remote.transport.netty.AbstractNettyEventListener exceptionCaught
경고: ExceptionEvent: local: /169.254.245.8:3782 remote: jw-PC/169.254.245.8:7000 :: java.io.IOException: 현재 연결은 원격 호스트에 의해 강제로 끊겼습니다
10월 18, 2014 7:27:06 오후 com.microsoft.wake.remote.transport.netty.AbstractNettyEventListener exceptionCaught
경고: ExceptionEvent: local: /169.254.245.8:7003 remote: /169.254.245.8:3779 :: java.io.IOException: 현재 연결은 원격 호스트에 의해 강제로 끊겼습니다
10월 18, 2014 7:27:06 오후 com.microsoft.reef.io.network.group.impl.ExceptionHandler onNext
심각: Exception occurred while processing a GroupComm operation caused by null
10월 18, 2014 7:27:06 오후 com.microsoft.reef.io.network.group.impl.ExceptionHandler onNext
심각: Exception occurred while processing a GroupComm operation caused by null
